src/Components/Notifications/NotificationsPills.tsx

```diff
@@ -24,7 +24,7 @@ export const NotificationsPills: React.FC = () => {
     { value: "All", name: "all" },
     hasPartnerOfferNotifications && { value: "Offers", name: "offers" },
     { value: "Alerts", name: "alerts" },
-    { value: "Following", name: "following" },
+    { value: "Follows", name: "following" },
```

Comment:
**nimor**: Should we update the name value as well? following -> follows

Reply:
Good idea! https://github.com/artsy/force/pull/13546